src/js/components/awardv2/idv/InfoTooltip.jsx

```diff
@@ -93,18 +96,21 @@ export default class InfoTooltip extends React.Component {
         }
         return (
             <div className="award__info-wrapper">
-                <div ref={(div) => {
-                    this.referenceDiv = div;
-                }}>
-                    <button
+                <div
+                    ref={(div) => {
+                        this.referenceDiv = div;
+                    }}>
+                    <div
+                        role="button"
+                        tabIndex="0"
                         onBlur={this.closeTooltip}
                         className="award__icon"
                         onFocus={this.showTooltip}
+                        onKeyPress={this.showTooltip}
                         onMouseEnter={this.showTooltip}
-                        onMouseLeave={this.closeTooltip}
```

Comment:
My main concern with moving the `onMouseLeave` from the icon to the tooltip removes the functionality of removing the tooltip when I leave the icon. With these changes I will now have to enter the tooltip div and then leave that div to remove the tooltip.

Reply:
Yep I changed this back actually.